config/manifests/bases/nfd-operator.clusterserviceversion.yaml

```diff
@@ -83,7 +83,7 @@ metadata:
     operators.operatorframework.io/builder: operator-sdk-v1.13.0+git
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/kubernetes-sigs/node-feature-discovery-operator
-  name: nfd-operator.v0.4.0
+  name: nfd-operator.v0.6.0
```

Comment:
master, or v0.6.0?

Reply:
same, this is `operator-sdk` related, won't matter